.github/workflows/build-and-publish.yml

```
@@ -74,15 +67,32 @@ jobs:
             SNAPSHOT="false"
           fi
 
-
-
           # Output the results
-          echo "VERSION=${VERSION}" >> $GITHUB_ENV
-          echo "SNAPSHOT=${SNAPSHOT}" >> $GITHUB_ENV
+          echo "TAG=${TAG}" >> $GITHUB_OUTPUT
           echo "version=${VERSION}" >> $GITHUB_OUTPUT
           echo "is-snapshot=${SNAPSHOT}" >> $GITHUB_OUTPUT
 
-          echo "Version: ${VERSION} Snapshot: ${SNAPSHOT} version=${VERSION} is-snapshot=${SNAPSHOT}"
+          echo "Version: ${VERSION} Snapshot: ${SNAPSHOT}"
+
+  build:
+    needs: parse-version
+    outputs:
+      version: ${{ needs.parse-version.outputs.version }}
+      is-snapshot: ${{ needs.parse-version.outputs.is-snapshot }}
+      rpm-version: ${{ steps.save-version.outputs.rpm-version }}
+      artifacts: ${{ steps.save-version.outputs.artifacts }}
+      rpm-artifacts: ${{ steps.save-version.outputs.rpm-artifacts }}
+      deb-artifacts: ${{ steps.save-version.outputs.deb-artifacts }}
+      zip-artifacts: ${{ steps.save-version.outputs.zip-artifacts }}
+      pkg-artifacts: ${{ steps.save-version.outputs.pkg-artifacts }}
+      sha-artifacts: ${{ steps.save-version.outputs.sha-artifacts }}
+      asc-artifacts: ${{ steps.save-version.outputs.asc-artifacts }}
+    runs-on: macos-13
```

```suggestion
    runs-on: macos-latest
```
If latest works it's nice to not have to update these when they are deprecated.